lxd/ip/vdpa.go

```diff
@@ -0,0 +1,423 @@
+package ip
+
+import (
+	"encoding/binary"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netlink/nl"
+)
+
+// vDPA Netlink name.
+const (
+	VDPAGenlName = "vdpa"
+)
+
+// vDPA Netlink command.
+const (
+	VDPACmdUnspec uint8 = iota
```

Comment:
@gabrielmougard another "general" code style comment here (I'll wait until you mark this ready for review before doing something more thorough). But generally I would suggest you unexport any variables/constants/functions unless they are actually needed to be used outside of the package. This helps to communicate to the reviewer what the external API is of your code. Thanks

Reply:
Looks like a lot of these constants are still unnecessarily exported? We shouldn't be exporting netlink concepts outside of the package.

Reply:
As I use `iota` and that the order matter (these are kernel bit flags on the netlink side), it might cause an issue. If the parameter is at the end of the `iota` block and is unused then it's ok, else it's not.. Or I just don't use `iota` to avoid all this.. @tomponline what do you think ?

Reply:
I dont see how that is relevant to them being exported or not though?

Reply:
By exported I mean https://go.dev/tour/basics/3